checker/app/controllers/ApiController.scala

```diff
@@ -42,7 +42,10 @@ class ApiController(
           }
         } recover {
         case e: Exception =>
-          InternalServerError(Json.obj("error" -> e.getMessage))
+          log.error(checkMarkers, "Error in /check", e)
```

Comment:
This is to ensure that any errors we receive are caught in our logs – ATM, we're just swallowing these errors from the POV of our logging.